### PR TITLE
Vanity link clarification plus some trivial cosmetic fixes on the Keyboard Shortcuts page

### PR DESCRIPTION
**Pull request by aervnu**
Flat out long list is still readable to some degree, but a collapsible is preferred. Would love it if someone can test the deployments out though

### CODE REVIEW

**Comment by itsWindows11** [comment]
Expand/collapse functionality doesn't seem to be working at all.

I had a better idea for this section, basically put each server and a brief description in a card, in a way that's consistent with the way VitePress features look in the home page.

You can deploy the site locally or test your changes by clicking on "Details" for the `deploy/netlify` test.

**Comment by itsWindows11** [comment]
Or simply use the details custom container, it has expand/collapse & you can set a custom title like any custom container used in the site.

https://vitepress.dev/guide/markdown#default-title

**Comment by aervnu** [comment]
> Or simply use the details custom container, it has expand/collapse & you can set a custom title like any custom container used in the site.
> 
> https://vitepress.dev/guide/markdown#custom-title

Yeah this seems to be the quickest and cleanest way. I'll deal with that, just a sec

File: src/mod/docs/index.md [hunk 1]
```diff
@@ -262,6 +262,38 @@ Bots will carry out these automated actions based on the following actions:
 Automatic actions performed by Cliptok will still trigger on messages that were blocked by Automod (see below).
 :::
 
+::: tip Server Invites
+Action will only be taken when there are unapproved invites. Certain approved invite links can be sent by any member regardless of their role (provided that they are not muted)
+:::
+
+::: details Approved vanity links
```
Comment (itsWindows11) [inline src/mod/docs/index.md:269]:
```suggestion
::: details Approved Vanity Links
```

File: src/mod/docs/index.md [hunk 2]
```diff
@@ -262,6 +262,36 @@ Bots will carry out these automated actions based on the following actions:
 Automatic actions performed by Cliptok will still trigger on messages that were blocked by Automod (see below).
 :::
 
+::: tip Server Invites
+Action will only be taken when there are unapproved invites. Certain approved invite links can be sent by any member regardless of their role (provided that they are not muted)
+:::
```
Comment (itsWindows11) [inline src/mod/docs/index.md:267]:
```suggestion
::: tip Note
Action will only be taken when unapproved invites are being sent. Certain approved invite links can be sent by any member regardless of their role, provided that they are not muted.
:::
```

**Review by itsWindows11 — CHANGES_REQUESTED** [review]

**Comment by aervnu** [comment]
My bad. I just resumed work on my new PC and while trying to sync all local repos to origin I accidentally clicked delete branch without realizing this PR is still open and hasn't been merged yet. Apologies.

**Comment by aervnu** [comment]
Fixed that. Would make the note less apparent though, since the text is now on a gray background instead of blue like the Tip container. But I do agree on the "see below" note - that one definitely needs to be relocated.

**Comment by FloatingMilkshake** [comment]
> Would make the note less apparent though, since the text is now on a gray background instead of blue like the Tip container

Honestly, my first thought was to shove the entire custom container into the collapsible, if possible—but I think I like the look of this. While the note doesn't draw as much attention, I think it's still probably prominent enough. So either way is okay with me!

Thanks!

**Review by FloatingMilkshake — APPROVED** [review]

**Pull request merged**